pkg/operator/credentialsrequest/credentialsrequest_controller.go

```diff
@@ -831,30 +836,30 @@ func (r *ReconcileCredentialsRequest) CreateOrUpdateOnCredsExist(ctx context.Con
 }
 
 func (r *ReconcileCredentialsRequest) updateActuatorConditions(cr *minterv1.CredentialsRequest, reason minterv1.CredentialsRequestConditionType, conditionError error) {
+	// Clear all existing conditions first
+	clearAllConditions(cr)
```

Comment:
I think we should revert to the original logic in `updateActuatorConditions`. The outcome of this switch case will be the same in terms of which conditions are set but we'll lose fields like `lastTransitionTime` should a condition remain the same.

Reply:
k reverting logic there